internal/test/wsu/wsu_test.go

```diff
@@ -111,6 +115,35 @@ func TestWSU(t *testing.T) {
 	t.Run("Network annotations were applied to node", testHybridOverlayAnnotations)
 	t.Run("HNS Networks were created", testHNSNetworksCreated)
 	t.Run("East-west networking", testEastWestNetworking)
+	t.Run("Check cni config generated on the Windows host", testCNIConfig)
+}
+
+// testCNIConfig tests if the CNI config has required hostsubnet and servicenetwork CIDR
+// TODO: split this into multiple tests when this grows
```

Comment:
Please create a ticket for this TODO, or track it in an existing ticket :)

Reply:
This is not a TODO but a NOTE as it is an indicator for the next person changing this to split it up if adding new tests.